.swiftformat

```diff
@@ -1,26 +1,27 @@
+# And operator &&
+--disable andOperator
+
 # Indent by 2 spaces
 --indent 2
 
-# Reference `self` explicitly
```

Comment:
`self` rule was causing some point free assignments to use `self.` too reference out of body functions, which caused compiler to not build...we can see if Swiftlint would be a better option here.